node-api/src/lib.rs

```diff
@@ -21,7 +21,6 @@ use alloc::{borrow::ToOwned, vec::Vec};
 use codec::{Decode, Encode};
 
 pub use decoder::*;
-pub use error::*;
```

Comment:
Needed to remove this one because in the api the node-api is also rexported with pub use ..::*; The default error should be the one from api-client, not the node-api.

Reply:
I see, in general I think we should be more conscious about exports anyhow than just wildcard re-exporting anything.

Reply:
https://github.com/scs/substrate-api-client/issues/430 :) Thanks for the input